PayPalCheckout/src/main/java/com/paypal/android/checkout/paymentbutton/error/Exceptions.kt

```diff
@@ -0,0 +1,7 @@
+package com.paypal.android.checkout.paymentbutton.error
+
+fun createFormattedIllegalArgumentException(enumName: String, enumValues: Int): IllegalArgumentException {
```

Comment:
I believe this function is only used within the SDK, could we make this function internal?